src/utils/awsLambdaLayerManager.ts

```diff
@@ -101,11 +102,22 @@ export class AwsLambdaLayerManager {
    * @returns Array of the published layers.
    */
   public async publishToAllRegions(): Promise<PublishedLayer[]> {
-    return await Promise.all(
-      this.awsRegions.map(region => {
-        return this.publishLayerToRegion(region);
+    const publishedLayers = await Promise.all(
```

Comment:
There is [`Promise.allSettled`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) beginning from Node 12.9.

This block would be much easier to follow (and more efficient) with the following code:
```js
const = publishedLayers = await Promise.all(
  this.awsRegions.map(
    async region => this.publishLayerToRegion(region).catch(error => logger.warn(`Something went wrong when publishing to AWS region ${region}: ${error.message}`))
  )
);
```